src/client/localplayer.cpp

```diff
@@ -879,7 +879,7 @@ void LocalPlayer::old_move(f32 dtime, Environment *env, f32 pos_max_d,
 	// TODO: This shouldn't be hardcoded but decided by the server
 	float player_stepheight = touching_ground ? (BS * 0.6f) : (BS * 0.2f);
 
-	v3f accel_f;
+	v3f accel_f(0, -gravity, 0);
```

Comment:
Should I perhaps not apply the fix to the `old_move` code?

Reply:
it's called old for a reason and supposed to be unchanging